ws-nextjs-app/pages/[service]/send/[id]/SubmitButton/styles.ts

```diff
@@ -0,0 +1,35 @@
+import { focusIndicatorThickness } from '#app/components/ThemeProvider/focusIndicator';
+import { Theme, css } from '@emotion/react';
+
+export default {
+  submit: ({ palette, fontSizes, fontVariants, spacings, mq }: Theme) =>
+    css({
+      backgroundColor: palette.SERVICE_NEUTRAL_CORE,
+      color: palette.WHITE,
+      width: '100%',
+      ...fontVariants.sansBold,
+      ...fontSizes.bodyCopy,
+      border: 'none',
+      padding: `${spacings.FULL + spacings.HALF}rem 0`,
+      marginBottom: `${spacings.DOUBLE}rem`,
+      cursor: 'pointer',
+      '&:hover': {
```

Comment:
I wonder if this should be hover and focus. (like '&:focus, &:hover'). I can't remember what the focus state is though we tend to apply the same styles to hover and focus states in the code base.

Reply:
yes good point ! the styling is the same for both, will update 👍